view/src/components/farms.js

```diff
@@ -358,6 +423,340 @@ class Farms extends Component {
+
+  /** Renders filtered produce results into Material-UI cards */
+  handleResultsRender = () => {
+    const { classes } = this.props;
+    const { filteredData } = this.state;
+
+    return (
+      <div>
+        <Grid container spacing={2} alignItem="center">
+          {filteredData.map((farm) => (
+            <Grid item xs={12} key={farm.farmId}>
+              <Card className={classes.root} variant="outlined">
+                <CardContent>
+                  <Typography variant="h5" component="h2">
+                    {farm.farmName}
+                  </Typography>
+                  {farm.farmTags.map((tag) => (
+                    <Chip className={classes.chip} label={tag} size="small" />
+                  ))}
+                  <Box
+                    display="flex"
+                    flexDirection="row"
+                    flexWrap="wrap"
+                    p={0}
+                    m={0}
+                  >
+                    <Box p={3}>
+                      <Typography className={classes.pos} color="textSecondary">
+                        Details:
+                      </Typography>
+                      <Typography variant="body2" component="p">
+                        Location of Farm: {`${farm.location.substring(0, 30)}`}
+                        <br />
+                        {`${farm.location.substring(30, 78)}`}
+                      </Typography>
+                    </Box>
+                    <Box p={3}>
+                      <Typography className={classes.pos} color="textSecondary">
+                        Point of Contact:
+                      </Typography>
+                      <Typography variant="body2" component="p">
+                        Name: {farm.contactName}
+                        <br />
+                        Phone: {farm.contactPhone}
+                        <br />
+                        Email: {farm.contactEmail}
+                      </Typography>
+                    </Box>
+                    <Box p={3}>
+                      <Typography className={classes.pos} color="textSecondary">
+                        Logistics:
+                      </Typography>
+                      <Typography variant="body2" component="p">
+                        Have Transportation Means:{" "}
+                        {farm.transportation ? "yes" : "no"}
+                        <br />
+                        Loading Dock: {farm.loadingDock ? "yes" : "no"}
+                        <br />
+                        Forklift: {farm.forklift ? "yes" : "no"}
+                      </Typography>
+                    </Box>
+                  </Box>
+                </CardContent>
+                <CardActions>
+                  {/* <Button size="small" color="primary" onClick={() => this.handleViewOpen({ farm })}>
+                                                {" "}
+                                                View{" "}
+                                            </Button> */}
```

Comment:
make sure to remove this comment.... (can always look at version history and add it back)